DataHub/Integration/03_ManageAttributeGroups.md

```diff
@@ -0,0 +1,305 @@
+
+
+## Create an attribute group
+
+Create an attribute group for new attributes you want to assign to a certain group.   
+
+#### Prerequisites
+
+Additional attributes for the attribute group have been created, see [Create an attribute](./01_ManageAttributes.md#create-an-attribute).
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTE GROUPS*
+
+![Attribute groups](../../Assets/Screenshots/DataHub/Settings/AttributeGroups/AttributeGroups.png "[Attribute groups]")
+
+1. Click the ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit) button to the right of the *Attribute groups* section in the left column.   
+  The *Edit attribute groups* view is displayed.
+
+  ![Edit attribute groups](../../Assets/Screenshots/DataHub/Settings/AttributeGroups/EditAttributeGroups.png "[Edit attribute groups]")
+
+2. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+  A plus sign is displayed in the attribute group column.
+
+  ![Add attribute group](../../Assets/Screenshots/DataHub/Settings/AttributeGroups/AddAttributeGroup.png "[Add attribute group]")
+
+3. Click the plus sign in the attribute group column.    
+  The *Add element* window is displayed in the attribute group column.
+
+  ![Add element](../../Assets/Screenshots/DataHub/Settings/AttributeGroups/AddElement.png "[Add element]")
+
+4. Enter a name for the attribute group in the *Name* field.
+
+5. Enter a key for the attribute group in the *Key* field. The number of characters is limited to 190.
+
+6. Click the [SAVE] button.   
+  The attribute group has been created. The *Add element* window is closed. The new attribute group is displayed in the attribute group column.
+
+  ![Attribute group created](../../Assets/Screenshots/DataHub/Settings/AttributeGroups/AttributeGroupCreated.png "[Attribute group created]")
+
+  > [Info] If you want to change the order of the attribute groups in the list, see [Change the order of attribute groups](#change-the-order-of-attribute-groups).
+
+7. Click the [SAVE] button in the upper right corner.  
+  The new attribute group has been saved. The *Edit attribute groups* view is closed.
+
+
+
+## Edit an attribute group
+
+After you have created an attribute group, you can edit it.
+You can create attribute sub-groups, change the order of attribute groups, add attributes to a group, change the order of attributes within a group, remove attributes from an attribute group or delete attribute groups.
+
+
+### Create an attribute sub-group
+
+Create an attribute sub-group to organize the attributes in special groups, for instance depending on product-related categories.
+If the attribute group already contained attributes, these attributes are automatically moved to first sub-group.
+
+Attribute groups are organized in a tree structure with a maximum depth of two levels. That means that a maximum of one sub-group level can be added to an attribute group.
+
```

Comment:
If the attribute group already contained attributes, these attributes are automatically moved to first sub-group.

Attribute groups are organized in a tree structure with a maximum depth of two levels. That means that a maximum of one sub-group level can be added to an attribute group.

Info fehlt im PIM? Wenn relevant, hinzufügen?